src/test/ui/issues/issue-37323.rs

```diff
@@ -1,4 +1,4 @@
-// build-pass (FIXME(62277): could be check-pass?)
+// check-pass
```

Comment:
Feel free to drop it, or add the flag, I don't know much about this test.

Reply:
@tmiasko Could you leave a FIXME for now? cc also @Xanewok

Reply:
I added the `-Zsave-analysis` to compile flags and left `check-pass` mode,
since I feel it is quite unlikely that we end up doing anything else here.